docs/navigation-parameters-reference.md

```diff
@@ -320,6 +323,11 @@ settings: {
     ```
 - **since**: 1.4.0
 
+### sideNavAccordionMode
+- **type**: boolean
+- **description**: overrides the default behaviour of categories whether multiple categories can be collapsed at the same time or similar to an accordion. (Default: false)
```

Comment:
```suggestion
- **description**: overrides the default behaviour of categories. When set to `true`, multiple categories can be collapsed at the same time and the navigation is similar to an accordion. 
- **default**: `false` 
```

Reply:
Hi @alexandra-simeonova , its actually the opposite. When set to true, only one category is open. The default functionality (false or not defined) is that any amount of collapsible categories can be open. :)

Reply:
```suggestion
- **description**: overrides the default behaviour of categories whether multiple categories can be collapsed. When set to `true`, only one category is collapsed. The navigation is similar to an accordion; when the user clicks another category the previously collapsed category is closed and the new one is opened.```